Lib/fontParts/base/anchor.py

```diff
@@ -392,5 +392,5 @@ def _round(self):
 
         Subclasses may override this method.
         """
-        self.x = normalizers.normalizeRounding(self.x)
-        self.y = normalizers.normalizeRounding(self.y)
+        self.x = round(self.x)
+        self.y = round(self.y)
```

Comment:
Not sure about using builtin round(). fontTools has fontTools.misc.fixedTools.otRound() which is more reasonable for coordinates. Py3 builtin round does some (I think) called banker's rounding, where values ending in .5 will be rounded to the nearest _even_ integer. Yes.

Reply:
Although (reading further) I see we've always used Py3 rounding. Hm. Maybe this is good for a separate issue.

Reply:
Yes, that's what we've done in the past, I do think we should use fontTool.misc.fixedTools.otRound for points (but not kerning?), but yes, a different issue

Reply:
Also kerning, pretty much everything. I find Py3's round() useless for most ordinary use, even though I understand it's importance for statistics and finance.

Reply:
Got it, issued a PR for that.